models/note.js

```diff
@@ -1,8 +1,16 @@
 const mongoose = require('mongoose');
 
 const noteSchema = new mongoose.Schema({
-  tite: String,
-  description: String
+  tite: {type: String, required: true},
+  description: {type: String, required: true},
+  author: {
+    id: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'user'
+        },
+    username: String,
+    required: true
+  }
```

Comment:
You don't need to save the whole author here, you can just save `authorId` here and later we'll just use it to reference it to the actual author.